bin/pulsar

```diff
@@ -214,6 +214,8 @@ if [ -z "$PULSAR_LOG_CONF" ]; then
     PULSAR_LOG_CONF=$DEFAULT_LOG_CONF
 fi
 
+OPTS="-agentlib:jdwp=transport=dt_socket,server=y,address=8000,suspend=n"
```

Comment:
any idea why do we need this?

Reply:
oops, I was debugging something and left this in.